src/components/Portal.ts

```diff
@@ -68,7 +68,7 @@ export class Portal extends LitElement {
         @bkdlocalechange=${this.handleLocaleChange.bind(this)}
       ></bkd-header>
       <bkd-content></bkd-content>
-      <bkd-footer></bkd-footer>
+      <bkd-footer currentLocale=${getLocale()}></bkd-footer>
```

Comment:
Da wir das nun an zwei Orten verwenden hier würde ich das in eine Variable tun:

```
render() {
  const currentLocale = getLocale();
  // ...
}
```

Mal schauen ob wir die Sprache auch noch an anderen Orten brauchen, sonst wäre es dann gut, wenn wir uns mal überlegen wie wir das zentral verfügbar machen könnten (Store oder so), damit wir das nicht überall reingeben müssen.